src/Compilers/CSharp/Test/Symbol/Symbols/ExtensionMethodTests.cs

```diff
@@ -4079,5 +4079,29 @@ void Validator(ModuleSymbol module)
 
             CompileAndVerify(source, validator: Validator, options: TestOptions.ReleaseDll);
         }
+
+        [Fact]
+        [WorkItem(65020, "https://github.com/dotnet/roslyn/issues/65020")]
+        public void ReduceExtensionsMethodOnReceiverTypeSystemVoid()
```

Comment:
>ReduceExtensionsMethodOnReceiverTypeSystemVoid [](http://example.com/codeflow?start=20&length=46)

Let's add similar test for VB

Reply:
Done

Reply:
It looks like the added test is not for the ```ReduceExtensionMethod``` API. The goal to have the test is to ensure that the API behaves consistently for both languages

Reply:
Ah OK. I added a semantic model test because the [check for System.Void](https://sourceroslyn.io/#Microsoft.CodeAnalysis.VisualBasic/Binding/Binder_Lookup.vb,1928) happens before the [call to ReduceExtensionMethod](https://sourceroslyn.io/#Microsoft.CodeAnalysis.VisualBasic/Binding/Binder_Lookup.vb,1220) I have changed the test to cover ReduceExtensionMethod directly